github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
@shurcooL I've checked how it works. When setting either `RequiredStatusChecks.IncludeAdmins`,`RequiredPullRequestReviews.IncludeAdmins` or `EnforceAdmins` will set all other properties to the same value - changing one changes all the rest. `RequiredPullRequestReviews.IncludeAdmins` and `RequiredPullRequestReviews.IncludeAdmins` are required fields in terms of request structure correctness (request fails with `422 Unprocessable Entity`, if they are not included - explained in previous comment), but logically they shouldn't be used since `EnforceAdmins` overrides them both. So, they are required, but deprecated by `EnforceAdmins`.